src/test/ui/type-alias-impl-trait/bounds-are-checked-2.stderr

```diff
@@ -0,0 +1,14 @@
+error[E0277]: the trait bound `T: std::clone::Clone` is not satisfied
+  --> $DIR/bounds-are-checked-2.rs:6:13
+   |
+LL | type X<T> = impl Clone;
+   |             ^^^^^^^^^^ the trait `std::clone::Clone` is not implemented for `T`
+   |
+help: consider restricting type parameter `T`
+   |
+LL | type X<T: std::clone::Clone> = impl Clone;
+   |         ^^^^^^^^^^^^^^^^^^^
```

Comment:
Re: my last comment. Oh, I see that you _are_ unifying their behavior. This will [very likely fail the crater run](https://github.com/rust-lang/rust/pull/69741#issuecomment-598973937).

Reply:
This is only for impl Trait.